test/e2e/framework/expect.go

```diff
@@ -31,21 +31,53 @@ func ExpectNotEqual(actual interface{}, extra interface{}, explain ...interface{
 }
 
 // ExpectError expects an error happens, otherwise an exception raises
+// Using ExpectErrorExplained is recommended
```

Comment:
```suggestion
// ExpectError checks that an error happened, otherwise the test fails.
//
// Should not be used anymore. Instead call ExpectErrorExplained
// with an explanation. That helps understand failures when the
// error is not descriptive enough by itself. Even when most errors
// that can occur are descriptive, that is hard to be certain about,
// so it's better to err on the side of caution and always provide
// an explanation.
```